legacy/setup.py

```diff
@@ -0,0 +1,48 @@
+import pathlib
+
+import setuptools
+
+library_root = pathlib.Path(__file__).parent
+
+with open(library_root / "README.md", "r") as readme:
+    long_description = readme.read()
+
+__version__ = ""
+with open(library_root / "conbenchlegacy" / "_version.py", "r") as f:
+    exec(f.read())  # only overwrites the __version__ variable
+
+
+def parse_requirements(path: str):
+    return [
+        line.strip()
+        for line in pathlib.Path(__file__)
+        .parent.joinpath(path)
+        .read_text()
+        .splitlines()
+    ]
+
+
+requirements_cli = parse_requirements(path="requirements-cli.txt")
+
+setuptools.setup(
+    name="conbenchlegacy",
+    version=__version__,
+    description="Continuous Benchmarking (CB) Framework Legacy Library and CLI",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    packages=setuptools.find_packages(),
+    entry_points={
+        "console_scripts": [
+            "conbench=conbenchlegacy.cli:conbench",
+        ],
+    },
```

Comment:
The new package name is currently `conbenchlegacy`, but this keeps the CLI name as `conbench` so installs and any imports will change slightly, but CLI calls will not.

We could call this package `conbench` and rename the main one to `conbenchwebapp` or whatever, but that would require changing a bunch of imports and would generally break a lot of things and be a lot more effort.